### PR TITLE
Starting the Rewrite of the Onboarding 🛥

### PR DESCRIPTION
**Pull request by benjdlambert**
### Problem
- The onboarding forms need sprucing up

### Solution
- Port the older ones to React

### Notes
@revett @stevenjack 
https://trello.com/c/J7lmLza2/298-cr-port-html-across-to-react

### CODE REVIEW

**Comment by revett** [comment]
https://trello.com/c/RkftX3VZ/325-fbi-add-targeting-data

**Comment by revett** [comment]
https://trello.com/c/RkftX3VZ/325-fbi-add-targeting-data

**Comment by revett** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;RkftX3VZ&#x2F;325-fbi-add-targeting-data"> [FBI] Add targeting data</a>

File: source/jsx/creator/form/fields/username.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import InputField from '../input-field';
+
+export default class Username extends React.Component {
+  validation(elem) {
+    const errors = [];
+    if (elem.value.length < 4) {
+      errors.push('Userbane must be 4 or more characters');
```
Comment (revett) [inline source/jsx/creator/form/fields/username.jsx]: Typo

**Comment by revett** [comment]
🍰 after typo

**Comment by benjdlambert** [comment]
I like `userbane`, let's just leave it like that.

**Comment by benjdlambert** [comment]
:)

**Comment by revett** [comment]
lol

**Comment by revett** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;J7lmLza2&#x2F;298-cr-port-html-across-to-react"> [CR] Port HTML across to React</a>

**Comment by revett** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;zlxcc8Fq&#x2F;299-cr-rewrite-existing-js-to-jsx"> [CR] Rewrite existing JS to JSX</a>

**Pull request merged**